src/components/Recipes/Recipes.tsx

```diff
@@ -3,13 +3,15 @@ import Card from '../UI/Card';
 import { FoodprintContext } from '../../store/foodprint-context';
 import { fetchData } from '../../utils/SpoonacularRequests';
 import { debounce } from '../../utils/Debounce';
+import { IngredientSearch } from '../../models/recipe';
 import List from './List';
 
 import "./Recipes.css";
 
 const Recipes: React.FC = () => {
   const foodprintCtx = useContext(FoodprintContext);
   const addRecipe: (title: string, id: number, image: string) => void = foodprintCtx.recipes.addRecipe;
+  const setRecipeSearchResults: (response: IngredientSearch[]) => void = foodprintCtx.recipeSearchResults.setRecipeSearchResults;
```

Comment:
this might have to go to a reducer check with @cguizar25

Reply:
but not a blocker for this PR, writing comments to come back to

Reply:
This can be incorporated into reducer down the line with a refactor